osu.Game/Screens/Play/ReplaySettings/PlaybackSettings.cs

```diff
@@ -1,27 +1,34 @@
 ﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
-using osu.Framework.Allocation;
-using osu.Game.Configuration;
-using osu.Framework.Graphics;
+using osu.Framework.Timing;
+using osu.Framework.Configuration;
 
 namespace osu.Game.Screens.Play.ReplaySettings
 {
     public class PlaybackSettings : ReplayGroup
     {
         protected override string Title => @"playback";
 
-        [BackgroundDependencyLoader]
-        private void load(OsuConfigManager config)
+        private readonly ReplaySliderBar<double> sliderbar;
+
+        public PlaybackSettings()
         {
-            Children = new Drawable[]
+            Child = sliderbar = new ReplaySliderBar<double>
             {
-                new ReplaySliderBar<double>
+                LabelText = "Playback speed",
+                Bindable = new BindableDouble(1)
                 {
-                    LabelText = "Playback speed",
-                    Bindable = config.GetBindable<double>(OsuSetting.PlaybackSpeed)
-                }
+                    MinValue = 0.5,
+                    MaxValue = 2
+                },
             };
         }
+
+        public void BindClock(IAdjustableClock clock)
+        {
+            var clockRate = clock.Rate;
+            sliderbar.Bindable.ValueChanged += rateMultiplier => clock.Rate = clockRate * rateMultiplier;

```

Comment:
this shouldn't be done in this function. it is dangerous if it was to be called more than once. do the bind in the ctor/load and only accept an incoming clock (and store to a local or whatever) here.